### PR TITLE
jail inactive validator with bad rating

### PR DESCRIPTION
**Pull request by sasurobert**

### CODE REVIEW

**Review by iulianpascalau — APPROVED** [review]
GJ! Will need to figure out how to test this

**Review by iulianpascalau — DISMISSED** [review]

**Review by raduchis — DISMISSED** [review]

**Review by raduchis — APPROVED** [review]

File: process/peer/process.go [hunk 1]
```diff
@@ -680,7 +700,7 @@ func (vs *validatorStatistics) setToJailedIfNeeded(
 		return
 	}
 
-	if peerAccount.GetUnStakedEpoch() == core.DefaultUnstakedEpoch && peerAccount.GetList() == string(core.InactiveList) {
+	if vs.isValidatorWithLowRating(peerAccount) {
```
Comment (SebastianMarian) [inline process/peer/process.go:703]: && peerAccount.GetList() == string(core.InactiveList), as it could be at this point already jailed

Comment (sasurobert) [reply process/peer/process.go:703]: That set does not matter. It is ok right now. In case of waiting or eligible there is an early return

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
System tests passed.

**Pull request merged**